.github/workflows/coverage.yml

```diff
@@ -33,6 +33,7 @@ jobs:
         run: |
             cargo install grcov
             sudo apt-get install lcov
+            cargo install cbindgen
```

Comment:
... and that 🙂 
```suggestion
```
I assume the other cbindgen removals should come when you update from `main`